samples/net/wifi/boards/esp32.conf

```diff
@@ -0,0 +1,18 @@
+CONFIG_WIFI=y
+CONFIG_WIFI_ESP32=y
+CONFIG_HEAP_MEM_POOL_SIZE=98304
+
+CONFIG_NETWORKING=y
+CONFIG_NET_L2_ETHERNET=y
+
+CONFIG_NET_IPV6=n
+CONFIG_NET_IPV4=y
+CONFIG_NET_DHCPV4=y
+
+CONFIG_NET_MGMT=y
+CONFIG_NET_MGMT_EVENT=y
+
+CONFIG_NET_LOG=y
+
+CONFIG_ESP32_WIFI_SSID="ssid"
+CONFIG_ESP32_WIFI_PASSWORD="password"
```

Comment:
Shouldn't ssid and password be passed to "connect" wifi_mgmt API? Why are those needed here?

Reply:
Not needed, removing.